src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -7359,13 +7359,14 @@ void CalcHeatBalanceInsideSurf2(EnergyPlusData &state,
 
     } // ...end of main inside heat balance DO loop (ends when Converged)
 
+    Real64 HConvIn;
```

Comment:
`HConvIn` is a local variable inside the loop?

Reply:
Yes. I just changed the declaration to inside the loop, so that it is not accidentally used elsewhere?